helm-service/controller/handler_base.go

```diff
@@ -58,6 +61,30 @@ func (h *HandlerBase) existsGeneratedChart(e keptnv2.EventData) (bool, error) {
 	return helm.DoesChartExist(e, genChartName, h.getConfigServiceURL())
 }
 
+func (h *HandlerBase) getCustomDeploymentURLs(chart *chart.Chart) (string, string) {
+	customDeploymentURIPublic := ""
+	customDeploymentURIlocal := ""
+
+	if chart.Values[CustomDeploymentURIPublicValuesProperty] != nil {
+		customDeploymentURIPublic = chart.Values[CustomDeploymentURIPublicValuesProperty].(string)
+	}
+
+	if chart.Values[CustomDeploymentURILocalValuesProperty] != nil {
+		customDeploymentURIlocal = chart.Values[CustomDeploymentURILocalValuesProperty].(string)
+	}
+
+	// sanity check on values
+	if customDeploymentURIPublic == "null" || customDeploymentURIPublic == "nil" {
+		customDeploymentURIPublic = ""
+	}
+
+	if customDeploymentURIlocal == "null" || customDeploymentURIlocal == "nil" {
+		customDeploymentURIlocal = ""
+	}
+
+	return customDeploymentURIPublic, customDeploymentURIlocal
```

Comment:
This should make the bot happy. Note that this appears to be the only function where you wrote `URIlocal` instead of `URILocal` and it doesn't match the sibling variable `customDeploymentURIPublic`.
```suggestion
	customDeploymentURILocal := ""

	if chart.Values[CustomDeploymentURIPublicValuesProperty] != nil {
		customDeploymentURIPublic = chart.Values[CustomDeploymentURIPublicValuesProperty].(string)
	}

	if chart.Values[CustomDeploymentURILocalValuesProperty] != nil {
		customDeploymentURILocal = chart.Values[CustomDeploymentURILocalValuesProperty].(string)
	}

	// sanity check on values
	if customDeploymentURIPublic == "null" || customDeploymentURIPublic == "nil" {
		customDeploymentURIPublic = ""
	}

	if customDeploymentURILocal == "null" || customDeploymentURILocal == "nil" {
		customDeploymentURILocal = ""
	}

	return customDeploymentURIPublic, customDeploymentURILocal
```